armv7-eabihf--glibc--stable-2018.02-2 toolchain returns error on signed overflow check.
---
 CompilerChecks.cmake | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CompilerChecks.cmake b/CompilerChecks.cmake
index 4d5ef02..63fa9b5 100644
--- a/CompilerChecks.cmake
+++ b/CompilerChecks.cmake
@@ -38,8 +38,6 @@ if (UNIX)
     add_c_compiler_flag("-Wuninitialized" SUPPORTED_COMPILER_FLAGS)
     add_c_compiler_flag("-Werror=uninitialized" SUPPORTED_COMPILER_FLAGS)
     add_c_compiler_flag("-Wimplicit-fallthrough" SUPPORTED_COMPILER_FLAGS)
-    add_c_compiler_flag("-Werror=strict-overflow" SUPPORTED_COMPILER_FLAGS)
-    add_c_compiler_flag("-Wstrict-overflow=2" SUPPORTED_COMPILER_FLAGS)
     add_c_compiler_flag("-Wno-format-zero-length" SUPPORTED_COMPILER_FLAGS)

     check_c_compiler_flag("-Wformat" REQUIRED_FLAGS_WFORMAT)
--
2.7.4
